indico/modules/categories/templates/management/base.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'layout/management_page.html' %}
+
+{% block banner_title %}
+    {{ category.title }}
+{% endblock %}
+
+{% block banner_actions %}
+    <div class="action-menu">
+        <div class="toolbar">
+            <div class="group">
+                <a class="i-button icon-transmission"
+                   title="{% trans %}Move category{% endtrans %}"
+                   href="#"></a>
```

Comment:
Not a big deal but to me `href` attribute is something that should be placed at the beginning of the element definition.